alfresco/231/overrides.gradle

```diff
@@ -1,8 +1,8 @@
-description = "Xenit Alfred API implementation Alfresco 7.4"
+description = "Xenit Alfred API implementation Alfresco 23.1.0"
 
 ext {
-    alfresco_version = '7.4.0'
-    alfresco_min_version = alfresco_version.substring(0, 3) + ".0"
+    alfresco_version = '23.1.0'
+    alfresco_min_version = alfresco_version.substring(0, 4) + ".0"
     //Not setting alfresco_max_version here to make it easier to test on next Alfresco version (only for latest version)
```

Comment:
See comment here: only for last version. Meaning you _do_ need to set alfresco_max_version here.